internal/configs/virtualserver.go

```diff
@@ -1078,7 +1078,7 @@ func (p *policiesCfg) addDosConfig(
 		}
 	}
 
-	p.Dos.ApDosAccessLogDest = dos.DosAccessLogDest
+	p.Dos.ApDosAccessLogDest = generateDosLogDest(dos.DosAccessLogDest)
```

Comment:
need to change to:
```
	if dos.DosAccessLogDest != "" {
		p.Dos.ApDosAccessLogDest = generateDosLogDest(dos.DosAccessLogDest)
	}
```

Reply:
Also in annotations.go in line 426:
`cfgParams.AppProtectDosAccessLogDst = appProtectDosAccessLogDst`
Need to change:
`cfgParams.AppProtectDosAccessLogDst = generateDosLogDest(appProtectDosAccessLogDst)
`

Reply:
I think these are fixed now.